_data/food.yml

```diff
@@ -11,6 +11,13 @@ websites:
       tfa: No
       twitter: maxhamburgare
 
+    - name: Morrisons
```

Comment:
Morrisons accounts holds bank details and addresses. [I'd say that fits 1 and 3.](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria)